account_invoice_validate_send_email/views/res_company_views.xml

```diff
@@ -12,6 +12,10 @@
                             name="invoice_mail_template_id"
                             domain="[('model', '=', 'account.move')]"
                         />
+                        <field
+                            name="receipt_mail_template_id"
+                            domain="[('model', '=', 'account.move')]"
+                        />
```

Comment:
I think it would be better if we could choose a mail template under Configuration > Settings, i.e., by adding this field in res.config.setting as a related field to the res.company's field.

Reply:
Can you give me a few more reasons why I should do it your way?  Now it seems to me that it is just more code.

Reply:
Now, if we want to set up or change template, we need to go to company record and change. 
In odoo standard design, most of the fields that are related with settings are under settings menu and make invisible them in company record view (eg, `currency_id`, `currency_exchange_journal_id` from `account` module).
In company record, we should only show the fields that are directed pointed for company infomration. (eg, vat, email, company social accounts).

Reply:
IMO, it's acceptable not to make fields available in `res.config.settings`; it adds more maintenance cost (the view adjustments are particularly cumbersome for this model) for a very small added value unless the module is a widely used one under OCA.

By the way, view adjustments on `res.config.settings` should be much easier in 17.0 as a result of https://github.com/odoo/odoo/pull/106425.